docs/reference/snapshot-restore/index.asciidoc

```diff
@@ -0,0 +1,87 @@
+[[snapshot-restore]]
+= Snapshot and restore
+
+[partintro]
+--
+
+// tag::snapshot-intro[]
+A snapshot is a backup taken from a running Elasticsearch cluster. You can take
+a snapshot of individual indices or of the entire cluster and store it in a
+repository on a shared filesystem, and there are plugins that support remote
+repositories on S3, HDFS, Azure, Google Cloud Storage and more.
+
+Snapshots are taken incrementally. This means that when it creates a snapshot of
+an index, Elasticsearch avoids copying any data that is already stored in the
+repository as part of an earlier snapshot of the same index. Therefore it can be
+efficient to take snapshots of your cluster quite frequently.
+// end::snapshot-intro[]
+
+// tag::restore-intro[]
+You can restore snapshots into a running cluster via the
+<<snapshots-restore-snapshot,restore API>>. When you restore an index, you can alter the
+name of the restored index as well as some of its settings. There is a great
+deal of flexibility in how the snapshot and restore functionality can be used.
+// end::restore-intro[]
```

Comment:
It may be worth explicitly pointing out _what_ you can restore from a backup, such as the full cluster or specific indices. The current writeup sort of implies a single index.

Reply:
Done.